test/e2e/antreapolicy_test.go

```diff
@@ -4399,14 +4397,19 @@ func TestAntreaPolicy(t *testing.T) {
 
 	t.Run("TestMulticastNP", func(t *testing.T) {
 		skipIfMulticastDisabled(t, data)
-		t.Run("Case=MulticastNPIGMPQueryAllow", func(t *testing.T) { testACNPIGMPQueryAllow(t, data) })
-		t.Run("Case=MulticastNPIGMPQueryDrop", func(t *testing.T) { testACNPIGMPQueryDrop(t, data) })
-		t.Run("Case=MulticastNPPolicyEgressAllow", func(t *testing.T) { testACNPMulticastEgressAllow(t, data) })
-		t.Run("Case=MulticastNPPolicyEgressDrop", func(t *testing.T) { testACNPMulticastEgressDrop(t, data) })
+		skipIfAntreaIPAMTest(t)
+		testMulticastNP(t, data, data.testNamespace)
 	})
 	k8sUtils.Cleanup(namespaces)
 }
 
+func testMulticastNP(t *testing.T, data *TestData, testNamespace string) {
+	t.Run("Case=MulticastNPIGMPQueryAllow", func(t *testing.T) { testACNPIGMPQueryAllow(t, data, testNamespace) })
+	t.Run("Case=MulticastNPIGMPQueryDrop", func(t *testing.T) { testACNPIGMPQueryDrop(t, data, testNamespace) })
+	t.Run("Case=MulticastNPPolicyEgressAllow", func(t *testing.T) { testACNPMulticastEgressAllow(t, data, testNamespace) })
+	t.Run("Case=MulticastNPPolicyEgressDrop", func(t *testing.T) { testACNPMulticastEgressDrop(t, data, testNamespace) })
+}
+
```

Comment:
If it's not going to be reused, I think not worth to extract this function

Reply:
It was reused in TestAntreaIPAMAntreaPolicy and TestAntreaPolicy.